strax/corrections.py

```diff
@@ -160,25 +183,60 @@ def write(self, correction, df, required_columns=('ONLINE', 'v1')):
         for req in required_columns:
             if req not in df.columns:
                 raise ValueError(f'Must specify {req} in dataframe')
-        # Compare against
+        # Compare against existing data
+        # We can change OFFLINE values in the past only if they are NaN
+        # We cannot change ONLINE values in the past
+        # We can add a new date(row) in the past(ONLINE) as long as it is the same value
```

Comment:
Can we  also get some developer documentation about this (not needed in this PR but in general)? CMT has become a very integral part and should be much better documented. Especially, about the jargon about ONLINE and OFFLINE corrections. How corrections should be implemented etc.

Reply:
done